python/prophet/models.py

```diff
@@ -13,6 +13,7 @@
 import pickle
 import pkg_resources
 import os
+from .numpyro_model import find_map, run_mcmc
```

Comment:
I think it's safer to move this into the `NumpyroBackend` class, since this file might be imported before the end user chooses their backend.